channel_type_test.go

```diff
@@ -13,6 +14,8 @@ func prepareChannelType(t *testing.T, c *Client) *ChannelType {
 	ct, err := c.CreateChannelType(ct)
 	require.NoError(t, err, "create channel type")
 
+	time.Sleep(6 * time.Second)
```

Comment:
I believe there was a latency/timing issue here, where the new ChannelType wasn't directly available to the test that created them

Reply:
possible